### DISCUSSION TITLE
CS6800i/WLW176i - Warm Water Stop Temperatures Not Displayed/Settable

### DISCUSSION THREAD

**Discussion by MarkusMit** [emsesp/EMS-ESP32#1852] [Q&A] [answered]
Hi, we just recently got a Bosch CS6800i heat pump, and last week I installed the EMS-ESP Gateway, so I just learning about everything. 

In general everthing looks good, but today I noticed that the stop temperatures ("Ausschalttemperatur") for all of the three "programs" (comfort 'wwcomfoff', eco 'wwecooff', eco+ 'wwecoplusoff') seem not to be working. (The 'selected temperatures' seem to be working fine.)
The set values are not displayed in the gateway's dashboard (either empty or zero) and they also can't be set through the dashboard
[emsesp_system_info.txt](https://github.com/user-attachments/files/16116884/emsesp_system_info.txt)

Let me know if there's something I could do to try and help debug the issue, please let me know, I'll try my best. :-)

**Comment by MichaelDvP** [comment, marked as answer]
Have you tried the dev build? I think these values are added from #1624

**Reply by MarkusMit** [reply]
Hi Michael, I tried today and it works. Thank you!
